src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -73,36 +73,30 @@ static ucs_status_t uct_rdmacm_cm_query(uct_cm_h cm, uct_cm_attr_t *cm_attr)
 
 static void uct_rdmacm_cm_handle_event_addr_resolved(struct rdma_cm_event *event)
 {
-    struct sockaddr    *remote_addr = rdma_get_peer_addr(event->id);
-    uct_rdmacm_cm_ep_t *cep         = (uct_rdmacm_cm_ep_t *)event->id->context;
-    char ip_port_str[UCS_SOCKADDR_STRING_LEN];
+    uct_rdmacm_cm_ep_t *cep = (uct_rdmacm_cm_ep_t *)event->id->context;
```

Comment:
if I'm not mistaken our code style requires to use the following format when casting
`(uct_rdmacm_cm_ep_t*)`